dominode-bootstrapper/dominode_bootstrapper/dominodeadmin.py

```diff
@@ -30,20 +35,32 @@ def bootstrap(
         minio_protocol: str = 'https'
 ):
     typer.echo('Bootstrapping DomiNode database...')
-    dbadmin.bootstrap(
-        db_user_name,
-        db_user_password,
-        db_name or db_user_name,
-        db_host,
-        db_port
-    )
+    try:
+        dbadmin.bootstrap(
+            db_user_name,
+            db_user_password,
+            db_name or db_user_name,
+            db_host,
+            db_port
+        )
+    except Exception as e:
+        typer.echo(f'Error: {e}...skipping')
+        pass
     typer.echo('Bootstrapping DomiNode minIO...')
-    minioadmin.bootstrap(
-        minio_access_key,
-        minio_secret_key,
-        minio_alias,
-        minio_host,
-        minio_port,
-        minio_protocol
+    try:
+        minioadmin.bootstrap(
+            minio_access_key,
+            minio_secret_key,
+            minio_alias,
+            minio_host,
+            minio_port,
+            minio_protocol
+        )
+    except Exception as e:
+        typer.echo(f'Error: {e}...skipping')
+        pass
+
+    geonodeadmin.bootstrap(
+
```

Comment:
Function parameters are missing here. They may be optional, but we still want to be able to specify them ;)